docs/visualization/path/response_time.md

```diff
@@ -87,3 +87,34 @@ plot.show()
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
+
+## TimeSeries
+
+The following scripts generate timeseries graphs of response time. Horizontal axis of the graph means timeseries and vertical axis means `Response time` in the best case, worst case, or all case.
```

Comment:
Perhaps Horizonal axis of the graph may mean `system time` ?

Reply:
I was a little confused because this graph visualizes Timeseries. But in other types of graphs, Horizonal axis is represented as timeseries...